go-controller/cmd/ovn-k8s-cni-overlay/app/helper_linux.go

```diff
@@ -0,0 +1,154 @@
+// +build linux
+
+package app
+
+import (
+	"fmt"
+	"net"
+	"os/exec"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/containernetworking/cni/pkg/skel"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"github.com/containernetworking/plugins/pkg/ip"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"github.com/vishvananda/netlink"
+)
+
+const defaultVethMTU = 1400
```

Comment:
I would add a TODO here so we move it to the CNI conf file later on and we don't forget about it. What do you think @rajatchopra ?

Reply:
I wanted to have the Linux code copy-paste from the original one without any changes in this patch. I'm gonna write a reminder for that.

Reply:
Yes we need to move this to config.